### ISSUE TITLE
MPR non-root test module pulls the wrong version of vaadin-lumo-styles

### ISSUE THREAD

**Issue by haijian-vaadin** [vaadin/flow#9822]
In the [mpr-test module](https://github.com/vaadin/multiplatform-runtime-internal/tree/master/mpr-tests), the `non-root` and `navigation` sub modules have IT test failures if the `~/.pnpm-store` has an older version (1.6.0) of the `vaadin-lumo-styles`. Seems the reason is that it pulls `vaadin-lumo-styles 1.6.0` from pnpm store, even though `1.6.1` is installed locally in `node_modules`.

To reproduce:
1. clone the [multiplatform-runtime-internal] (https://github.com/vaadin/multiplatform-runtime-internal/tree/master/mpr-tests)
2. Make sure you have a `vaadin-lumo-styles 1.6.0` in your `~/.pnpm-store` folder, which could be done by running a Vaadin 18.0.0 project in `pnpm` mode (the default).
3. Go to the `non-root` submodule, run `mvn jetty:run`, go to localhost:9998/context, you will see the error as in the attached screenshot.
![Screenshot 2021-01-21 at 9 28 19](https://user-images.githubusercontent.com/30408303/105317615-12fd2200-5bcb-11eb-946b-54e8acb4e947.png)

**Comment by caalador** [comment]
I couldn't reproduce this. Can you test with latest snapshot?
Also isn't the module missing the vaadin-maven-plugin definition as I am not getting the widgetset
```
<plugin>
  <groupId>com.vaadin</groupId>
  <artifactId>vaadin-maven-plugin</artifactId>
</plugin>
```
Another question is also does it need to run in dev mode or could it do a production build?
This would be just to add:
```
<plugin>
  <groupId>com.vaadin</groupId>
  <artifactId>flow-maven-plugin</artifactId>
</plugin>
```

**Comment by haijian-vaadin** [comment]
Seems the bug is fixed by [this commit](https://github.com/vaadin/flow/commit/9a0db2c52b68b27fece9eb31c5bc0db8a3474d58). Verified that it was reproducible one commit before, but no longer reproducible since the commit.

But just spotted another problem. After clicking the `Go to other` button, I got js errors on the page, see the attached screenshot, wonder if I should create another ticket for this issue, or should we reuse the same.
![Screenshot 2021-01-25 at 15 38 54](https://user-images.githubusercontent.com/30408303/105713640-0f97cc80-5f24-11eb-82ce-de4ea7230e41.png)
